timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/rdf/CollectionDescription.java

```diff
@@ -52,17 +56,22 @@ public static CollectionDescription createCollectionDescription(String entityTyp
   }
 
   public static CollectionDescription createCollectionDescription(String entityTypeName, String vreName,
+                                                                  String rdfUri, boolean isUnkown) {
+    return new CollectionDescription(createEntityTypeName(entityTypeName, vreName), vreName, rdfUri, isUnkown);
+  }
+
+  public static CollectionDescription createCollectionDescription(String entityTypeName, String vreName,
                                                                   String rdfUri) {
-    return new CollectionDescription(createEntityTypeName(entityTypeName, vreName), vreName, rdfUri);
+    return new CollectionDescription(createEntityTypeName(entityTypeName, vreName), vreName, rdfUri, false);
   }
 
   public static CollectionDescription getAdmin(Vertex archetypeVertex) {
     String entityTypeName = archetypeVertex.value(ENTITY_TYPE_NAME_PROPERTY_NAME);
-    return new CollectionDescription(entityTypeName, "Admin", createRdfUri(entityTypeName));
+    return new CollectionDescription(entityTypeName, "Admin", createRdfUri(entityTypeName), false);
```

Comment:
I'm confused. How can we be sure that getAdmin never results in the "concepts" collection?